output/install.nsi

```diff
@@ -332,6 +334,8 @@ Section "Uninstall"
   ; Remove registry keys
   DeleteRegKey HKLM SOFTWARE\Rime
   DeleteRegKey HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Weasel"
+  ; Remove autorun in HKCU
+  DeleteRegValue HKCU "Software\Microsoft\Windows\CurrentVersion\Run" "WeaselServer"
```

Comment:
Should this line be placed after `SetRegView 64`?

Reply:
seems there are no "WOW6432Node" in hkcu

so no need to set reg view